cmd/kube-apiserver/app/testing/testserver.go

```diff
@@ -209,8 +219,9 @@ func StartTestServer(t Logger, instanceOptions *TestServerInstanceOptions, custo
 		server.GenericAPIServer.StorageVersionManager = instanceOptions.StorageVersionWrapFunc(server.GenericAPIServer.StorageVersionManager)
 	}
 
-	errCh := make(chan error)
+	errCh = make(chan error)
 	go func(stopCh <-chan struct{}) {
```

Comment:
do you know why do we have to capture the stopCh in this goroutine?

Reply:
well - it's probably not needed...

You want me to change this?

Reply:
nope, it was a question for me, for learning ...